lib/ptexter/package.json

```diff
@@ -34,6 +34,24 @@
             "{pkg}/phead-lines.py"
           ]
         }
+      },
+      "wc-l": {
+        "binary": {
+          "artifact": {
```

Comment:
What do you think of creating single package with single requirements.txt, but different entrypoints for it?